### PR TITLE
chore: Remove deprecations for v1.9.0

### PR DESCRIPTION
**Pull request by spydon**
# Description

Removes everything that was marked as deprecated and that should be removed for v1.9.0.

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [x] No, this PR is not a breaking change.

## Related Issues

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

File: packages/flame/lib/src/camera/viewfinder.dart [hunk 1]
```diff
@@ -169,8 +169,8 @@ class Viewfinder extends Component
 
   @override
   void onGameResize(Vector2 size) {
-    super.onGameResize(size);
     _initZoom();
+    super.onGameResize(size);
```
Comment (erickzanardo) [inline packages/flame/lib/src/camera/viewfinder.dart:173]: I am curious on why the change in the order?

**Review by erickzanardo — APPROVED** [review]

File: packages/flame/lib/src/camera/viewfinder.dart (hunk 1 above)
Comment (spydon) [reply packages/flame/lib/src/camera/viewfinder.dart:173]: If you have children of the viewfinder, you might want to know the new zoom from your parent when the child is running `onGameResize`. (Same thing with viewport, where it actually caused some bugs)

Comment (erickzanardo) [reply packages/flame/lib/src/camera/viewfinder.dart:173]: Maybe we should document that?

Comment (spydon) [reply packages/flame/lib/src/camera/viewfinder.dart:173]: Doesn't the user expect that the parent has the correct value? Or how do you mean document it? :sweat_smile:

Comment (erickzanardo) [reply packages/flame/lib/src/camera/viewfinder.dart:173]: I mean adding some kind of docs that if you want the change the zoom on the on game resize, that you might want to call the super after calculating the new values, I mean, now that you explained to me it is pretty obvious, but at first I didn't realized it haha

Comment (spydon) [reply packages/flame/lib/src/camera/viewfinder.dart:173]: This is the core viewfinder, on a normal component you can do what you want and it will still work, and if you subclass the viewfinder you might want to handle zoom differently and you definitely know what you're doing if you've gone that deep. So I don't think there is anything obvious to document. 😅
